client/src/Reviews/Review.jsx

```diff
@@ -0,0 +1,67 @@
+import React, { useState, useEffect } from 'react';
+
+const Review = ({ review }) => {
+  const [showMore, setShowMore] = useState(false);
+
+  const handleShowMore = () => {
+    setShowMore(!showMore);
+  };
+
+  // display first 250 characters of review body
+  // refactor have only one div tag
+  const checkReviewLength = () => {
+    if (review.body.length <= 250) {
+      return (
+        <div className="review-body">{review.body}</div>
+      );
+    } else if (!showMore) {
+      return (
+        <div className="review-body">
+          {review.body.slice(0, 250)}...
+          <br></br>
+          <a onClick={handleShowMore}>Show more</a>
+        </div>
+      );
+    } else {
+      return (
+        <div className="review-body">
+          {review.body}
+          <br></br>
+          <a onClick={handleShowMore}>Show less</a>
+        </div>
+      );
+    }
+  };
+
+  const style = { // just to help visualize for now
+    border: '1px solid black'
+  };
+
+  return (
+    <div style={style} className="review-tile">
+      <div>
+        <span className="stars">stars go here</span>
+        {/* don't think it's possible to reference sales with the API */}
```

Comment:
Yeah, I think this is one of the impossible requests. The BRD references the user's email but that's not something we have access to